src/psyclone/virtual_utils.py

```diff
@@ -48,5 +48,6 @@ def within_virtual_env():
     :rtype: bool
     '''
     import sys
+    # pylint: disable=no-member
```

Comment:
My pylint says this suppression is 'useless' but I'm happy to let that go otherwise we're into the vagaries of which precise version of pylint you're running.

Reply:
I tested again, and I don't get the useless message, but do get a warning when I remove this warning. For the record: pylint 1.9.4 (python 2.7) and 2.3.1 (python 3) show the same behaviour: no 'useless' message, but a warning if I remove the disable statement. So I leave this in (and might considering updating my python 2 and 3 pylints up-to-date).